src/main/java/org/ohdsi/circe/cohortdefinition/PayerPlanPeriod.java

```diff
@@ -19,8 +19,10 @@
 package org.ohdsi.circe.cohortdefinition;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.ohdsi.analysis.versioning.CdmVersion;
 import org.ohdsi.circe.vocabulary.Concept;
 
+@CdmVersion(range = ">=5.3")
```

Comment:
Could we also apply a @cdmVersion(range = ">=5.0") on the base criteria class? Will that be overridden here?  I suggest this because I think it makes sense to declare that all criteria have some sort of `@CdmVersion` attached to them.

Reply:
You were typing it at the same time as I was commiting the change. Look below